openedx/core/djangoapps/user_api/tests/test_views.py

```diff
@@ -617,7 +619,7 @@ def test_login_form(self):
                 "placeholder": "",
                 "instructions": "",
                 "restrictions": {
-                    "max_length": password_max_length(),
+                    "max_length": 5000,
```

Comment:
Guh, so now this magic constant is in two places? You should really consider making it a named constant that you import. For one example, so you can have that nice comment, which should be here too, in only one place.